Pod/Classes/WPInputMediaPickerViewController.m

```diff
@@ -73,85 +65,6 @@ - (void)setupMediaPickerViewController {
 
 - (void)viewDidLayoutSubviews {
     [super viewDidLayoutSubviews];
-    [self configureCollectionView];
-}
-
-- (void)configureCollectionView {
-    CGFloat photoSpacing = 1.0f;
-    CGFloat photoSize;
-    CGSize cameraPreviewSize;
-    UICollectionViewFlowLayout *layout = (UICollectionViewFlowLayout *)self.mediaPicker.collectionView.collectionViewLayout;
-    if (self.scrollVertically) {
-        CGFloat frameWidth = self.view.frame.size.width;
-        NSUInteger numberOfPhotosForLine = [self numberOfPhotosPerRow:frameWidth];
-
-        photoSize = [self.mediaPicker cellSizeForPhotosPerLineCount:numberOfPhotosForLine
-                                                       photoSpacing:photoSpacing
-                                                         frameWidth:frameWidth];
-
-        // Check the actual width of the content based on the computed cell size
-        // How many photos are we actually fitting per line?
```

Comment:
What happened to this code? I don't see this section in the updated `WPMediaPickerCollectionView`?

Reply:
I realised that we didn't need to do this any longer, because the way we calculate the number of rows.